src/pages/recipes/events/index.md

```diff
@@ -0,0 +1,139 @@
+---
+keywords:
+  - UXP Events
+  - Event Scripting
+---
+# Events
+This section shows how to work with InDesign UXP events.
+
+## Overview
+InDesign can respond to standard application and document events, such as opening a file, creating a new one, printing, and importing text and graphic files from a disk, etc. Scripts can be attached to events using the eventListener scripting object. Scripts that use events are the same as other scripts except that after the event listener is associated, the script is expected to be blocked by a promise and wait for the event in the background until the event listener is removed. Then the promise can be resolved to exit the script. Once you run the Events Scripts with an event listener, they are called back automatically after the corresponding event occurs, rather than being run by the user (from the Scripts palette).
```

Comment:
Indesign 'responds' or 'emits' these events?

Reply:
It would be good to explain the use case where it is necessary to attach to such events.

Reply:
Done